src/dialog/overflow-menu/overflow-menu-pane.component.ts

```diff
@@ -86,10 +85,26 @@ export class OverflowMenuPane extends Dialog {
 				event.preventDefault();
 				listItems[listItems.length - 1].focus();
 				break;
+
+			case "Esc": // IE specific value
+			case "Escape": {
+				event.stopImmediatePropagation();
+				this.doClose();
+				break;
+			}
+		}
+	}
+
+	clickClose(event) {
+		if (this.dialogConfig.parentRef.nativeElement.firstChild.contains(event.target) ||
+			this.listItems().some(button => button === (event.relatedTarget)) ||
+			(event.type === "focusout" && event.relatedTarget === this.dialogConfig.parentRef.nativeElement)) {
```

Comment:
What are the brackets for?
```suggestion
			event.type === "focusout" && event.relatedTarget === this.dialogConfig.parentRef.nativeElement) {
```